bin/.travis/test.sh

```diff
@@ -96,15 +96,20 @@ docker run -ti --rm ez_php:latest-dev bash -c "php -v; php -m"
 printf "\Integration: Behat testing on ez_php:latest and ez_php:latest-dev with eZ Platform\n"
 cd volumes/ezplatform
 
-export COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/redis.yml:doc/docker/selenium.yml" SYMFONY_ENV="behat" SYMFONY_DEBUG="0" PHP_IMAGE="ez_php:latest" PHP_IMAGE_DEV="ez_php:latest-dev"
+export COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/redis.yml:doc/docker/selenium.yml" SYMFONY_ENV="behat" SYMFONY_DEBUG="0" 
 
 if [ -f doc/docker/install-dependencies.yml ]; then
+    # Use dev variant for installation as it contains mysqldump
+    export PHP_IMAGE="ez_php:latest-dev"
```

Comment:
Shouldn't it be `PHP_IMAGE_DEV `?

Reply:
Platform v2 uses PHP_IMAGE for installation: https://github.com/ezsystems/ezplatform/blob/2.5/doc/docker/install-dependencies.yml#L6
This is a change from v1, which used PHP_IMAGE_DEV for it: https://github.com/ezsystems/ezplatform/blob/1.13/doc/docker/install.yml#L16

and that's the reason for the changes above.

Reply:
OK 😀